Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM quay.io/openshift/origin-must-gather:4.6
 
-# Save original gather script
-RUN mv /usr/bin/gather /usr/bin/gather_original
+# Save original gather script and make our script executable
+RUN mv /usr/bin/gather /usr/bin/gather_original && \
+    chmod +x gather_istio.sh
```

Comment:
Do this on the final file instead (/usr/bin/gather/gather_istio.sh) below

Reply:
done

Reply:
yeah, thanks - I need coffee :)